easy_tensorflow/tf_functions.py

```diff
@@ -229,8 +229,7 @@ def train_tensorflow(sess, trX, trY, train_steps, full_train, train_size, net_ty
     predict_op = py_x
```

Comment:
Here are some recommendations for changes:

1. **Docstrings**: Some functions are missing docstrings, such as `decode_classifications()`. Including a brief description of the function, its parameters, and its output can greatly improve code readability and maintainability.

2. **Error Handling**: In the `train_tensorflow()` function, the generic exception is caught but not re-raised or logged. This could make troubleshooting difficult in the future. It's recommended to log the exception or re-raise it after printing the error message.

3. **Consistency**: There is inconsistency in the use of single and double quotes for strings. It's recommended to stick to one style for consistency.

4. **Variable Naming**: Some variable names are not descriptive, such as `p` in the `predict()` method of `Classifier` and `Regresser` classes. Use more descriptive variable names to improve code readability.

5. **Code Comments**: Some parts of the code, especially complex logic, could benefit from additional comments to explain what is happening.

6. **Code Duplication**: The `predict()` methods in `Classifier` and `Regresser` classes are almost identical. This code duplication could be avoided by creating a base class with a `predict()` method that both `Classifier` and `Regresser` inherit from.

7. **Use of Magic Numbers**: In the `train_tensorflow()` function, the number 100 is used in the for loop. It's recommended to replace these with a named constant to improve code readability and maintainability.